### DISCUSSION TITLE
Equality on write

### DISCUSSION THREAD

**Discussion by Faaatman** [isar/isar#347] [Help] [answered]
Hello,
I'm using isar to cache my app data. upon retrieving the data from the backend do I have to check for equality before writing again or does isar do it for me? keep in mind I'm using `replaceOnConflict: true`

Thank you.

**Comment by simc** [comment, marked as answer]
What do you mean? If you `put` an object with an existing id, the existing entry will be overwritten. It doesn't matter whether the objects are equal or not.

**Reply by Faaatman** [reply]
Ok perfect👍🏻
Thank you🙏🏼
